core/src/main/java/com/arcrobotics/ftclib/geometry/Vector2d.java

```diff
@@ -154,6 +154,16 @@ public Vector2d scale(double scalar) {
         return new Vector2d(m_x * scalar, m_y * scalar);
     }
 
+    /**
+     * Takes the vector and changes the length while maintaining the direction
+     *
+     * @param value number to normalize vectors with
+     * @return a normalized vector
+     */
+    public Vector2d normalize(double value) {
+        return scale(value / magnitude());
+    }
```

Comment:
Why is the normalization happening with an input?
Normalization is typically defined as scaling all elements of the vector such that the dot product with itself is 1.
I think that the method should be
```java
public Vectoe2d norm() {
  return scale(1 / magnitude());
}
```
Can you explain further as to why there is a parameter here?

Reply:
That's not normalization.